### PR TITLE
Fix internal issue 15 public private forx

### PR DESCRIPTION
**Pull request by jannesblobel**
add group properties

### CODE REVIEW

File: source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx [hunk 1]
```diff
@@ -236,10 +236,14 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 					repo: args.routeParams.repository,
 				})
 				.then((response) => {
+					telemetryBrowser.group("repository", gitOrigin, {
+						visibility: response.data.private ? "Private" : "Public",
+						isFork: response.data.fork ? "Fork" : "isNotFork",
+						parentGitOrigin: response.data.parent?.git_url.replace("git://", ""),
```
Comment (samuelstroschein) [inline source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx:242]: why don't you use parseOrigin for the gitorigin?

You can plug the url into `parseOrigin({remotes: [response.data.parent.git_url]})`

Comment (jannesblobel) [reply source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx:242]: parseOrigin required remote="origin"  and the URL 
[{remote: 'origin', url: response.data.parent.git_url}]
and we also need to adjust the function.  response.data.parent.gut_url return git://github.com/inlang/example.git
We are not covering git:// in this function. 
That's the reason.

Comment (jannesblobel) [reply source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx:242]: It looks clearer and more readable if we do this with the split function.

Comment (samuelstroschein) [reply source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx:242]: then change `parseOrigin` to only take the origin aka `parseOrigin(origin)`. 

We should use parseOrigin to ensure that the origin is always identical.

Comment (jannesblobel) [reply source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx:242]:
```suggestion
							? parseOrigin({ remotes: [{ remote: "origin", url: response.data.parent.git_url }] })
```
If we rewrite the function, I have to handle the remote on a other place
```suggestion
							? parseOrigin({ remotes: [{ remote: "origin", url: response.data.parent.git_url }] })
```

Comment (jannesblobel) [reply source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx:242]: Therefore, I would set the "remote" like suggested,  if we want to use the same function

Comment (samuelstroschein) [reply source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx]: Okay sounds good.

**Pull request merged**